src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -468,21 +468,22 @@
         [:div {:style {:padding "22px 48px 40px" :backgroundColor (:background-light style/colors)}}
          content
          (when (or show-cancel? ok-button)
-           [:div {:style {:marginTop 40 :textAlign "center"}}
+           [:div {:style {:marginTop (if ok-button 40 25) :textAlign "center"}}
```

Comment:
If only the Cancel button is present, with no Done button, the margin is smaller.